frontend/www/js/omegaup/components/course/StudentProgress.vue

```diff
@@ -120,6 +121,27 @@ export default class StudentProgress extends Vue {
     );
   }
 
+  get weightedMean(): Array<number> {
+    const totalPoints =
+      this.assignments
+        .map((assignment) => assignment.max_points)
+        .reduce((a, b) => (a ?? 0) + (b ?? 0)) ?? 0;
```

Comment:
```suggestion
        .reduce((acc, curr) => acc + (curr ?? 0), 0);
```

recuerda siempre agregarle el segundo parámetro a [`reduce()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) para evitar todos los `undefined`s.

Reply:
Listo.